docs/concepts/data-management/filtering/index.mdx

```diff
@@ -135,6 +158,14 @@ To filter releases, keep the following in mind:
 - Globbing rules apply and there is no special casing for SemVer. This allows for matching prefixes, such as `my-example@1.*`.
 - The filter never applies to events without a release.
 
+### Log Message
+
+To use inbound data filters for log messages, keep the following in mind:
+
+- You can provide multiple patterns, one per line. The filter applies if any of the patterns match.
+- On logs, the filter matches the log message in the format. We do not recommend matching the full log message, and suggest you match with wildcards. For example, to match any "Connection timeout asbq33q", use the filter `*Connection timeout*`. The wildcard matcher can be used at the beginning or end of the string.
```

Comment:
Might be good to mention that log message uses glob patterns in the `Glob Matching` section below.